README.md

```diff
@@ -1,58 +1,173 @@
+| Data files                      | [![yes]](https://www.getzola.org/documentation/templates/overview/#load-data) | ![yes] | ![yes] | ![no]   |
+| LiveReload                      | [![yes]](https://www.getzola.org/documentation/getting-started/cli-usage/#serve) | ![no]  | ![yes] | ![yes]  |
+
+**Notes:**
+
+- *Multilingual site*
+  - zola gets ![ehh] because despite having a basic translations system, it is being redesigned to evade its current limitations (see [discussion on the forum](https://zola.discourse.group/t/rfc-internationalization-system-rework/546))
+- *Search*
+  - zola gets ![ehh] because search engine is disabled by default in some languages, as explained [in the docs](https://www.getzola.org/documentation/content/multilingual/#configuration)
+
+### Content library
+
+|                                 | [Zola](https://getzola.org/)   | [Cobalt](https://cobalt-org.github.io/) | [Hugo](https://gohugo.io/)   | [Pelican](https://blog.getpelican.com/) |
+|:--------------------------------|:------:|:------:|:------:|:-------:|
 | Aliases                         | ![yes] | ![no]  | ![yes] | ![yes]  |
-| Pagination                      | ![yes] | ![no]  | ![yes] | ![yes]  |
-| Custom taxonomies               | ![yes] | ![no]  | ![yes] | ![no]   |
-| Search                          | ![yes] | ![no]  | ![no]  | ![yes]  |
-| Data files                      | ![yes] | ![yes] | ![yes] | ![no]   |
-| LiveReload                      | ![yes] | ![no]  | ![yes] | ![yes]  |
-| Netlify support                 | ![yes] | ![no]  | ![yes] | ![no]   |
-| Vercel support                  | ![yes] | ![no]  | ![yes] | ![yes]  |
+| Assets co-location              | [![ehh]](https://www.getzola.org/documentation/content/overview/#asset-colocation) | ![yes] | ![yes] | ![yes]  |
+| HTML shortcodes                 | [![yes]](https://www.getzola.org/documentation/content/shortcodes/) | ![no]  | ![yes] | ![yes]  |
+| Format-specific shortcodes      | [![yes]](https://www.getzola.org/documentation/content/shortcodes/) |   ?    |    ?   |    ?    |
+| Internal links                  | [![ehh]](https://www.getzola.org/documentation/content/linking/) | ![no]  | ![yes] | ![yes]  |
+| Warn or error for broken links  | [![no]](https://github.com/getzola/zola/issues/977#issuecomment-759725671)  | ![no]  | [![yes]](https://gohugo.io/content-management/cross-references/) | ? |
+| Custom taxonomies               | [![yes]](https://www.getzola.org/documentation/content/taxonomies/) | ![no]  | ![yes] | ![no]   |
+| Multiple static mounts          | [![no]](https://github.com/getzola/zola/issues/499) | ![no] | [![yes]](https://gohugo.io/hugo-modules/configuration/#module-config-mounts) | ? |
+
+**Notes:**
+
+- *Assets co-location*
+  - zola receives ![ehh] because assets cannot be shared across pages/sections, despite living in the same folder ([discussion on the forum](https://zola.discourse.group/t/reusing-markdown-docs-from-github-repo-in-zola-site/776))
```

Comment:
how does cobalt/hugo solve that to get a `yes`? I can't find asset co-location in Cobalt docs

Reply:
Not sure. That was simply copied over from the current README.